frontend/src/app/lightning/nodes-per-country-chart/nodes-per-country-chart.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="full-container h-100">
+
+  <div class="card-header">
+    <div class="d-flex d-md-block align-items-baseline" style="margin-bottom: -5px">
+      <span i18n="lightning.nodes-per-country">Lightning nodes per country</span>
+      <button class="btn p-0 pl-2" style="margin: 0 0 4px 0px" (click)="onSaveChart()">
+        <fa-icon [icon]="['fas', 'download']" [fixedWidth]="true"></fa-icon>
+      </button>
+    </div>
+    <small style="color: #ffffff66" i18n="lightning.tor-nodes-excluded">(Tor nodes excluded)</small>
+  </div>
+
+  <div class="container pb-lg-0 bottom-padding">
+    <div class="pb-lg-5">
+      <div class="chart w-100" echarts [initOpts]="chartInitOptions" [options]="chartOptions"
+        (chartInit)="onChartInit($event)">
+      </div>
+    </div>
+
+    <div class="text-center loadingGraphs" *ngIf="isLoading">
+      <div class="spinner-border text-light"></div>
+    </div>
+
+    <table class="table table-borderless text-center m-auto" style="max-width: 900px">
+      <thead>
+        <tr>
+          <th class="text-left rank" *ngIf="!isMobile()" i18n="mining.rank">Rank</th>
```

Comment:
Function calls from template can cause performance problems so consider setting a static variable instead

Reply:
Yeah actually I already hide them with css but I forgot to remove those